src/lib/network_pool/pool_sink.ml

```diff
@@ -95,22 +95,18 @@ module Base
         if BC.is_expired cb then Deferred.return None
         else
           let summary = `String (Diff.summary @@ Envelope.Incoming.data env) in
-          [%log debug] "Verifying $diff from $sender"
-            ~metadata:
-              [ ("diff", summary)
-              ; ("sender", Envelope.Sender.to_yojson env.sender)
-              ] ;
+          let metadata =
+            [ ("diff", summary)
+            ; ("sender", Envelope.Sender.to_yojson env.sender)
+            ]
+          in
+          [%log debug] "Verifying $diff from $sender" ~metadata ;
           match
             Rate_limiter.add rl env.sender ~now:(Time.now ())
               ~score:(Diff.score env.data)
           with
           | `Capacity_exceeded ->
-              [%log debug]
-                ~metadata:
-                  [ ("sender", Envelope.Sender.to_yojson env.sender)
-                  ; ("diff", summary)
-                  ]
-                "exceeded capacity from $sender" ;
+              [%log debug] ~metadata "exceeded capacity from $sender" ;
```

Comment:
I have used these logs a lot when debugging missed/lost transactions. I'd prefer if we can include at least fee payer part of a transaction.